addon/components/o-s-s/alert.stories.js

```diff
@@ -36,6 +36,26 @@ export default {
         defaultValue: { summary: '' }
       },
       control: { type: 'text' }
+    },
+    plain: {
+      description: 'When false, a white background color is displayed',
```

Comment:
```suggestion
      description: 'When plain is true, a gray background color is displayed, otherwise a white one',
```
Maybe more clear because default value is true